ci-operator/config/openshift/ocs-operator/OWNERS

```diff
@@ -1,18 +0,0 @@
-# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```

Comment:
Just double checking that there are no other files present in the dir if the `OWNERS` file is being removed.

Reply:
Yup, dir's gone.